client/src/index.js

```diff
@@ -122,24 +122,34 @@ class MainView extends React.Component {
     this.setState(state => ({
       surveyPanelIsHidden: !state.surveyPanelIsHidden,
     }))
+
     if (this.state.surveyPanelIsHidden) {
       this.setState({ surveyButtonAnimation: true })
       setTimeout(() => {
         this.setState({ surveyButtonAnimation: false })
       }, 3000)
     }
+
+    if (window.innerWidth < 500 && !this.state.commentPanelIsHidden) {
```

Comment:
Make this a variable, so all window.innerWidth to => isMobileViewport or something, so easier to change 500 later on, replace in all places.